docs/packages/pkg/restinio.rst

```diff
@@ -0,0 +1,33 @@
+.. spelling::
+
+    restinio
+
+.. index:: networking ; restinio
+
+.. _pkg.restinio:
+
+restinio
+========
+
+-  `Official <https://github.com/Stiffstream/restinio>`__
+-  `Example <https://github.com/cpp-pm/hunter/blob/master/examples/restinio/CMakeLists.txt>`__
+
+.. code-block:: cmake
+
+    find_package(Threads REQUIRED)
+
+    hunter_add_package(asio)
+    find_package(asio CONFIG REQUIRED)
+
+    hunter_add_package(http-parser)
+    find_package(http-parser CONFIG REQUIRED)
+
+    hunter_add_package(fmt)
+    find_package(fmt CONFIG REQUIRED)
+
+    hunter_add_package(restinio)
+    find_package(restinio CONFIG REQUIRED)
+    target_link_libraries(... restinio::restinio)
+
+The reason you must manually bring in restinio's dependencies is because
```

Comment:
dependencies are normally done by the hunterized package. The user of the package shouldn't need to specify dependencies by hand. Please hunterize restinio package instead following the documentation

https://hunter.readthedocs.io/en/latest/creating-new/create/cmake-dependencies.html

Reply:
As my comment in the docs notes, restinio is an INTERFACE library. Therefore, its cmake dependencies get exported with the target.

restinio is fully hunterised, see https://github.com/ned14/restinio/blob/master/CMakeLists.txt. This isn't the problem here.